packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -9,11 +9,19 @@ import '../ui/elements.dart';
 import '../ui/fake_flutter/dart_ui/dart_ui.dart';
 import '../ui/flutter_html_shim.dart';
 import '../utils.dart';
+import 'package:mutex/mutex.dart';
```

Comment:
move this import to the section with other package: imports.
I can't wait till the IntelliJ can 100% automatically curate the imports section.

Reply:
Done